.circleci/continue_config.yml

```diff
@@ -2747,27 +2747,25 @@ jobs:
       - setup_docker:
           docker_image: datadog/dd-trace-ci:php-compile-extension-alpine-<< parameters.php_major_minor >>
       - run:
-          name: Build extension
-          command: |
-            # Backtraces on Alpine are apparently sketchy, and removed in Alpine 3.17:
-            # https://gitlab.alpinelinux.org/alpine/aports/-/merge_requests/36722
-            # Ensure libexecinfo is _not_ installed, so support isn't detected
-            # by config.m4
-            alpine_version=$(cat /etc/os-release | awk -F= '$1 == "VERSION_ID" { print $2 }')
-            alpine_major_version=$(echo $version | cut -d . -f 1)
-            alpine_minor_version=$(echo $version | cut -d . -f 2)
-            if [ $alpine_major_version -eq 3 ] && [ $alpine_minor_version -lt 17 ] ; then
-                apk update && apk del libexecinfo libexecinfo-dev
-            fi
```

Comment:
Just curious: why is this not needed anymore?

Reply:
Because we're now on alpine 3.18